### DISCUSSION TITLE
Postal address for Direct QSLs

### DISCUSSION THREAD

**Discussion by spREGR** [wavelog/wavelog#855] [Q&A] [answered]
Hi, 
I have an XML subscription of QRZ.com. In Wavelog, I can see the label printing option only prints the QSL details. The bureau we have in India, sends QSL cards once a year and hence, I would like to send few dozen cards Direct. Is there a way, I can look up the addresses. I searched but couldn't find how to do it. I believe this feature is not available. Can it be done via Wavelog - say running some SQL query or calling a functional via command line?
73
de VU2TUM

**Comment by HB9HIL** [comment, marked as answer]
Hello @spREGR 

This was requested already several times. Problem here is the law and that it's may illegal to store automatically addresses of other OM's on your local system. 

The fastest way to get the address for a direct QSL card to use the QRZ.com shortcut button in the QSL Queue

<img width="332" alt="Bildschirmfoto 2024-09-01 um 13 25 33" src="https://github.com/user-attachments/assets/e2c6c4ca-9d57-416c-9153-d07bf8f9102d">
